cmake/pcre2-config.cmake.in

```diff
@@ -34,7 +34,7 @@ if("@REQUIRE_PTHREAD@") # REQUIRE_PTHREAD
   find_dependency(Threads)
 endif()
 
-include("${CMAKE_CURRENT_LIST_DIR}/@TARGETS_EXPORT_NAME@.cmake")
+include("${CMAKE_CURRENT_LIST_DIR}/pcre2-targets.cmake")
```

Comment:
can we please change the this PR into a draft, until we had ironed all testing issues and the proposed code is therefore expected to work?

Reply:
Done.